scripts/regenerate.sh

```diff
@@ -99,6 +99,14 @@ Mgrpc/testing/test.proto=google.golang.org/grpc/interop/grpc_testing,\
 Mgrpc/testing/payloads.proto=google.golang.org/grpc/interop/grpc_testing,\
 Mgrpc/testing/empty.proto=google.golang.org/grpc/interop/grpc_testing
 
+# test feature use separate service package and only test one proto file
```

Comment:
This is our script to regenerate the proto packages we maintain; it is not intended for testing.  Can we find another way to test this?

Reply:
I have no idea where the goldenfile should be placed.

1. How about putting it in `cmd/protoc-gen-go-grpc/testdata/routeguide_grpc/route_guide_grpc.pb.go`, adding a generation script in `cmd/protoc-gen-go-grpc/regenerate.sh`, and `creating cmd/protoc-gen-go-grpc/doc.go` with a `go:generate` directive?

2. Alternatively, remove the goldenfile and use:
```
grep 'routeguide "google.golang.org/grpc/examples/route_guide/routeguide"' \
${TEMPDIR}/examples/route_guide/routeguide/route_guide_grpc.pb.go
```

Reply:
I removed the goldenfile and used grep to check.